src/components/pages/Classroom/FeedbackBadges.js

```diff
@@ -1,24 +1,22 @@
 import React from 'react';
-import { Card, Avatar, Badge } from 'antd';
+import { Card, Avatar } from 'antd';
 
 const { Meta } = Card;
 
-const FeedbackBadges = ({ badges }) => {
+const FeedbackBadges = ({ badge }) => {
```

Comment:
I like the naming change here! Could also apply it to the component itself, changing FeedbackBadges to FeedbackBadge as well for consistency.